scss/modules/content-types.scss

```diff
@@ -354,28 +358,48 @@
                 .num {
                     color: $grayLight;
                     padding-right: 15px;
+                    display: table-cell;
+                    width: 1%; // make cell fit content
```

Comment:
Well, the difference for me is that already one letter is probably longer than 1px, this cannot be guaranteed for percentages.